<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>f.tea.p</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Local Directory Tree</string>
    </property>
    <widget class="QTreeView" name="tree_l">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>171</height>
      </rect>
     </property>
     <property name="indentation">
      <number>5</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_up_local">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_enter_local">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Remote Directory Listing</string>
    </property>
    <widget class="QListView" name="list_r">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_enter_remote">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_up_remote">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QPushButton" name="btn_retr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RETR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>STOR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dele">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DELE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rmd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RMD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_mkd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>MKD</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_rename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>255</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>new name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_apply_rn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>255</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Abdullah AKALIN - 141180001</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
